### PR TITLE
Fix usage of `repository` and `npx` in package.json

### PR DESCRIPTION
**Pull request by webpro**
The `repository` field did originally not support `directories`, but now we can add them, which is in this PR (also see https://docs.npmjs.com/cli/v9/configuring-npm/package-json#repository).

Since package managers should add `bin` directory to the path automatically, there's no need to put `npx` in between when running scripts.

### CODE REVIEW

**Review by alber70g — APPROVED** [review]

**Review by Randynamic — APPROVED** [review]

**Review by ash-vd — APPROVED** [review]

**Comment by webpro** [comment]
So this PR requires a lot of approvals, since it touches a lot of packages. Unfortunately github does not provide an efficient/optimal list of approvals to unblock this PR, but I think it's this:

```
/packages/libs/chainweb-stream-client           @Takadenoshi
/packages/tools/create-kadena-app               @jermaine150
```

By the way, I think packages should have at least two owners, but that'll probably be resolved once we install teams on github.

**Review by Takadenoshi — APPROVED** [review]

**Comment by webpro** [comment]
Oh the joy of changing something in almost each and every package, I missed one:

```
/packages/libs/react-components                 @eileenmguo @sstraatemans @isaozler
```

**Review by sstraatemans — APPROVED** [review]

**Pull request merged**
